test/iam.ts

```diff
@@ -144,20 +147,20 @@ describe('IAM', () => {
   });
 
   describe('testPermissions', () => {
-    it('should throw an error if permissions are missing', () => {
-      assert.throws(() => {
-        iam.testPermissions(util.noop);
-      }, /Permissions are required\./);
-    });
+    // it('should throw an error if permissions are missing', () => {
```

Comment:
same for testPermissions function,i think this test case is not required if we giving type to iam variable.
https://github.com/praveenqlogic/nodejs-pubsub/blob/master/src/iam.ts#L313